src/TweetProcessor.js

```diff
@@ -120,12 +126,18 @@ class TweetProcessor {
 
   _push_to_msg_queue (muser, roomid, tweet, type) {
     var time = Date.parse(tweet.created_at);
+    let content;
+    try {
+      content = this.tweet_to_matrix_content(tweet, type)
+    } catch (e) {
```

Comment:
`e` get's lost, better log it at least (I'm thinking future breaking changes of `tweet_to_matrix_content` would be hard to debug).